pipelines/deployment.yml

```diff
@@ -1158,8 +1183,6 @@ jobs:
         EQ_URL: 'https://eq-test/session?token='
         JSON_SECRET_KEYS: ((ci_json_secret_keys))
         JWT_SECRET: ((ci_jwt_secret))
-        OAUTH_CLIENT_ID: 'ons@ons.gov'
-        OAUTH_CLIENT_SECRET: 'password'
         SECRET_KEY: ((ci_enrolement_code_encryption_key))
 
 - name: django-oauth2-test-ci-deploy
```

Comment:
Do the existing django-oauth2 jobs need to be removed from here now? Or are they staying as they are until the service is removed completely?

Reply:
Keeping them in until it's removed completely. Will have a clean up once we fully deprecate the django-oauth-test   repo.